config/prow/config.yaml

```diff
@@ -483,6 +483,10 @@ branch-protection:
               protect: true
             release-1.22:
               protect: true
+            release-1.23:
+              protect: true
+            release-1.24:
+              protect: true
```

Comment:
```suggestion
```
There is no `release-1.24` branch yet, technically `main` is 1.24 at the moment, the branch protection is for the `release-1.23` branch that was created a few days ago

Reply:
That makes sense, thanks for that!